package.json

```diff
@@ -77,6 +75,7 @@
     "prettier": "3.0.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "rimraf": "5.0.1",
```

Comment:
Where was this needed?

Reply:
This one is for cleaning up the `<root_dir>lib/*` directory. It basically cleans up that folder before every build in "clear" script